docs/UserGuide.adoc

```diff
@@ -686,12 +686,13 @@ Receive reminders for deadlines and events the next day.
 |  `*mod* view INDEX` +
    `*mod* view MOD_CODE`
 
-.5+|Task
+.6+|Task
```

Comment:
Can help change the task add the /task to /desc

Reply:
yah got it